README.md

```diff
@@ -1,24 +1,180 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# 📗 Table of Contents
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Hello Rails React ](#-hello-rails-react-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage ](#usage-)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
 
-Things you may want to cover:
+<!-- PROJECT DESCRIPTION -->
 
-* Ruby version
+# 📖 Hello Rails React <a name="about-the-project"></a>
+**Hello Rails React** is an implemention of a connection between Ruby on Rails back-end and React front-end by displaying a random greeting message.
 
-* System dependencies
+## 🛠 Built With <a name="built-with"></a>
 
-* Configuration
+### Tech Stack <a name="tech-stack"></a>
 
-* Database creation
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://github.com/microverseinc/linters-config/tree/master/ror">Linters</a></li>
 
-* Database initialization
+  </ul>
+</details>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">ROR</a></li>
+  </ul>
+</details>
 
-* How to run the test suite
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
 
-* Services (job queues, cache servers, search engines, etc.)
+<!-- Features -->
 
-* Deployment instructions
+### Key Features <a name="key-features"></a>
 
-* ...
+- Display a random greeting message
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- N/A
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+- [Ruby on Rails](https://rubyonrails.org/) installed and running. To get more information, read the [installation guide](https://guides.rubyonrails.org/).
+
+- [PostgreSQL](https://www.postgresql.org/) installed and running
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+  git clone git@github.com:blase147/hello-rails-react.git
+  cd Hello-Rails-React
+```
+
+### Install
+
+Install thr project dependencies with:
+
+```
+  bundle install
+  rails javascript:install:webpack
+  rails db:create
+  rails db:migrate
+  rails db:seed
```

Comment:
The app installation fails at the seeding of the database with the error in the attached image. Kindly look into this error.

<img width="419" alt="image" src="https://user-images.githubusercontent.com/27709832/235442803-df292bde-7c8a-4cab-86db-69236670e110.png">